net/unbound/Makefile

```diff
@@ -43,7 +43,8 @@ define Package/unbound
 endef
 
 define Package/unbound/description
-  This package contains the Unbound daemon.
+  This package contains the Unbound daemon. 
+  Please see https://github.com/openwrt/packages/blob/master/net/unbound/files/README.md
```

Comment:
I'm fine with extending the description, but not so much with it only being a link, much less to a non-upstream document.